### DISCUSSION TITLE
Option to open tab as child adjacent to parent tab

### DISCUSSION THREAD

**Discussion by rieje** [piroor/treestyletab#3465] [Q&A]
When I create a new tab, it's created as a sibling. This is the desired behavior, except in one case: sometimes I want to create it as a child tab at the top (i.e. adjacent to the parent tab). 

Is it possible to keep the existing behavior has default but also have a hotkey-bound to open link/tab as child tab? It probably makes sense for this child tab to get added as the last child (below), but when there are a ton of existing child tabs I prefer the tab be created at the top next to the parent tab. I tend to group tabs by their content and go through them in a LIFO manner hence this preference.

**Comment by piroor** [comment]
On such cases I use drag-and-drop of a link into the sidebar. TST opens a link at the dropped position.

For hard-to-drag cases (like trackpad) currently no option to do that. TST More Tree Commands https://github.com/piroor/tst-more-tree-commands looks to be a good place to implement such commands as extra context menu commands.
